tests/generation/test_utils.py

```diff
@@ -2755,24 +2754,9 @@ def test_speculative_sampling_target_distribution(self):
 
 @pytest.mark.generate
 @require_torch
-class GenerationIntegrationTests(unittest.TestCase, GenerationIntegrationTestsMixin):
```

Comment:
`GenerationIntegrationTestsMixin` contained integration tests shared across frameworks. They were moved here. A few redundant tests were removed in the process (e.g. `max_new_tokens` checks, which are part of the mixin tests)

Reply:
> They were moved here

You mean those tests are moved to this class in this PR, right?

Reply:
Moved from `GenerationIntegrationTestsMixin` [deleted] to `GenerationIntegrationTests `, yes :)